python/tvm/relay/frontend/tflite.py

```diff
@@ -1030,4 +1153,6 @@ def from_tflite(model, shape_dict, dtype_dict):
     outputs = [exp_tab.get_expr(get_tensor_name(subgraph, i)) for i in model_outputs]
     outputs = outputs[0] if len(outputs) == 1 else _expr.Tuple(outputs)
     func = _expr.Function(analysis.free_vars(outputs), outputs)
-    return _module.Module.from_expr(func), params
+    mod = _module.Module.from_expr(func)
+    mod = tvm.relay.qnn.transform.CanonicalizeOps()(mod)
```

Comment:
@zhiics @vinx13 @tqchen Should we remove this from here and make it a part of build_module.cc?